helpers/slices/strings.go

```diff
@@ -0,0 +1,37 @@
+package slices
```

Comment:
I'm not a fan of `utility` packages, so if anyone has a better suggestion on where to put this/name it fire away.